src/groupeddataframe/splitapplycombine.jl

```diff
@@ -1483,10 +1515,13 @@ end
            copycols::Bool=true, keepkeys::Bool=true, ungroup::Bool=true)
 
 Apply `args` to `gd` following the rules described in [`combine`](@ref) and return the
-result as a `DataFrame` if `ungroup=true` or `GroupedDataFrame` if `ungroup=false`.
+result as a `DataFrame` if `ungroup=true` or `GroupedDataFrame` if `ungroup=false`
+(in this case the returned value retains the order of groups of `gd`).
```

Comment:
I'd put this detail below: the first paragraph should just explain what's the purpose of the function.

Reply:
OK